src/Microsoft.ML.StandardLearners/Standard/Online/doc.xml

```diff
@@ -13,8 +13,8 @@
         and an option to update the weight vector using the average of the vectors seen over time (averaged argument is set to True by default).
       </remarks>
     </member>
-    <example>
-      <example name="OGD">
+    <example name="OGD">
```

Comment:
>example name="OGD" [](start = 5, length = 18)

out of curiosity, is order of having name / not having name matter? In AP example you have <example><example name=""> and here you have <example name=""><example>

Reply:
it depends how you include the node in the code. 

see the long answer to Pete's comment. Should clarify it. 
Note the * notation:

<include file='../Microsoft.ML.StandardLearners/Standard/doc.xml' path='doc/members/member[@name="OLS"]/*' />


---
In reply to: [205926135](https://github.com/dotnet/machinelearning/pull/592#discussion_r205926135) [](ancestors = 205926135)